joomla/joomla-cms
# [com_joomlaupdate] Extend the allowed respose codes for updates

Pull Request for Issue #11806 
### Summary of Changes

This allows the folowing Resposecodes if you update:

```
$allowedResponseCodes = array(
    // HTTP 200 Ok
    '200',
    // HTTP 301 Moved Permanently
    '301',
    // HTTP 302 Found
    '302',
    // HTTP 303 See Other
    '303',
);
```
### Testing Instructions
- Install an old version of joomla like 3.6.1
- apply this patch
- install the joomla update
- all should work as before
### Documentation Changes Required

None.
